spec/models/registration_spec.rb

```diff
@@ -40,10 +40,11 @@
           location = build(:location, :supported)
           registration = build(:registration, zipcode: location.zipcode)
 
-          registration.save
+          saved = registration.save
           user = registration.user
           location = registration.location
 
+          expect(saved).to be true
```

Comment:
You can also do `expect(registration).to be_persisted`, without setting `saved`. Up to you!